docs/resources/recognition.md

```diff
@@ -0,0 +1,46 @@
+# Recognizing Contributions
+
+There are a variety of ways that we would like to recognize community contributions to hackweeks. 
+
+## Website Presence
+
+We invite all people who join us in organizing and hosting a hackweek to be included in our "Meet the team" section of each hackweek website. 
+
+We use this YAML markdown template for each team member. 
+
+```
+- title: 
+  avatar: 
+  role: 
+  organizations:
+  - name: 
+    url: 
+  - name: 
+  bio: 
+  expertise:
+  - 
+  - 
+  social:
+  - icon: github
+    icon_pack: fab
+    link: 
+  user_groups:
+  - 
+  ```
+
+The first step is to copy the text above into your favorite editor and add your information. Here is an example of how the text you include gets rendered on the website:
+
+  ```{image} ../images/website-bio.jpg
+:alt: example of how to fill out website bio
+:width: 1100px
+:align: center
+```
+
+Note that it is important to precisely follow the indentation and spacing layout shown here, so that the website correctly interprets the YAML format. Also, we encourage everyone to use their GitHub avatar as the source of their personal photograph.
+
+## Uploading to the website
+
+There are two ways you can upload your information so that we can include you on the website:
+
+1. send the completed YAML file to one of the lead organizers, and they will push it to the website

```

Comment:
I'd prefer to remove this first option :) I think encouraging the pull request route is best to push tutorial leads towards getting comfortable with github

Reply:
Agreed.